recipes/bezier/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bezier" %}
+{% set version = "0.2.1" %}
```

Comment:
Sorry @synapticarbors, been traveling. I'll try to get to it today. Thanks a lot for checking in on me!